packages/graphiql-react/src/theme.tsx

```diff
@@ -53,4 +63,17 @@ export function useTheme() {
   return useMemo(() => ({ theme, setTheme }), [theme, setTheme]);
 }
 
+type ThemeContextProviderProps = { children: ReactNode };
+
+export function ThemeContextProvider(props: ThemeContextProviderProps) {
+  const theme = useThemeInternal();
+  return (
+    <ThemeContext.Provider value={theme}>
+      {props.children}
+    </ThemeContext.Provider>
+  );
+}
+
+export const useTheme = createContextHook(ThemeContext);
```

Comment:
can you add a documentation block above, for the api docs?

Reply:
Added